### PR TITLE
Add new ffnet-0.8.3.ebuild and ffnetui-0.8.3.2.ebuild

### PR DESCRIPTION
**Pull request by mrkwjc**
I added new ffnet-0.8.3.ebuild and i modified ffnet metadata (sourceforge --> github). Also i added new ffnetui package which is GUI for ffnet. Please merge...

### CODE REVIEW

File: dev-python/ffnet/ffnet-0.8.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```
Comment (jlec) [inline dev-python/ffnet/ffnet-0.8.3.ebuild]: Please use ```EAPI=6```

File: dev-python/ffnet/ffnet-0.8.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+PYTHON_COMPAT=( python2_{6,7} python3_{3,4} )
+
+inherit distutils-r1 flag-o-matic fortran-2 toolchain-funcs
+
+DESCRIPTION="Feed-forward neural network for python"
+HOMEPAGE="http://ffnet.sourceforge.net/"
+SRC_URI="http://pypi.python.org/packages/source/f/ffnet/${P}.tar.gz"
```
Comment (jlec) [inline dev-python/ffnet/ffnet-0.8.3.ebuild]: Please use ```repoman```. see ```qgrep pypi```for examples.

File: dev-python/ffnet/ffnet-0.8.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+src_compile() {
+	distutils-r1_src_compile ${FCONFIG}
+}
+
+python_install_all() {
+	distutils-r1_python_install_all
+	if use examples; then
+		insinto /usr/share/doc/${PF}
+		doins -r examples
+	fi
```
Comment (jlec) [inline dev-python/ffnet/ffnet-0.8.3.ebuild]: Just do 
```ebuild
use examples && dodoc -r examples
```

File: dev-python/ffnet/metadata.xml [hunk 4]
```diff
@@ -7,12 +7,12 @@
...
   ffnet is a fast and easy-to-use feed-forward neural
-  network training solution for python.
+  network training library for python.
 </longdescription>
   <use>
     <flag name="matplotlib">Use matplotlib for drawing</flag>
   </use>
   <upstream>
-    <remote-id type="sourceforge">ffnet</remote-id>
+    <remote-id type="github">ffnet</remote-id>
```
Comment (jlec) [inline dev-python/ffnet/metadata.xml]: This isn't the whole tag.

File: dev-python/ffnetui/ffnetui-0.8.3.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 flag-o-matic toolchain-funcs
```
Comment (jlec) [inline dev-python/ffnetui/ffnetui-0.8.3.2.ebuild]: flag-o-matic toolchain-funcs aren't used.

Comment (jlec) [inline dev-python/ffnetui/ffnetui-0.8.3.2.ebuild]: Could you please drop the ```flag-o-matic toolchain-funcs``` here?

**Comment by jlec** [comment]
@mrkwjc Could you please squash your commits and rebase onto master?

**Comment by mrkwjc** [comment]
Travis fails but it seems this is not ffnet fault.

**Comment by jlec** [comment]
Looks good to merge @SoapGentoo do you take care?

**Comment by SoapGentoo** [comment]
@jlec on it

**Comment by SoapGentoo** [comment]
Merged

**Pull request closed without merging**